runtime/src/global_request_scheduler_minheap.c

```diff
@@ -74,8 +75,7 @@ sandbox_request_get_priority_fn(void *element)
  * Initializes the variant and registers against the polymorphic interface
  */
 void
-global_request_scheduler_minheap_initialize()
-{
+global_request_scheduler_minheap_initialize() {
```

Comment:
Breaks style.

Reply:
Sorry, I think I changed the breaks style when I traced the code, the original sledge code wasn't like this. I will recover it.